### ISSUE TITLE
PR: _canoncial_uri example

### ISSUE THREAD

**Issue by twMat** [Jermolene/TiddlyWiki5#2826]
**PR**
```
title: Motovun Jack _canonical_uri
tags: picture
_canonical_uri: https://c1.staticflickr.com/7/6217/6292279493_0824a351d7_b.jpg
tiddler-type: image/jpeg
```

**Notes** (not part of the PR)
* The image is from the source provided in the [Motovun Jack.jpg image](http://tiddlywiki.com/#Motovun%20Jack.jpg) example.
* The proposed title:  The other example pictures on tw.com follow a `name.filetype` format. This is understandable given that this is the default title when one imports an image *but* it risks confusing viewers into thinking that the title should include the image format. I note that also the ascii example follows this format which I think is a bad idea (because that is not a filetype, or is it?). This is why I propose the title `Motovun Jack _canonical_uri`. Maybe the arbitrariness of the title should be made even clearer like `Motovun Jack _canonical_uri example`

**Comment by Marxsal** [comment]
To me, the _canonical_uri dangling on the end is somewhat confusing. Maybe it could have a little more structure.  Maybe something like: 

Example of _canonical_uri use -- Motovun Jack

or

The 'Motovun Jack' image as an example of _canonical_uri use

I'm not sure if any of this would help the original poster (in the forum) since he wants to use local, relative pathways.

**Comment by twMat** [comment]
Yeah... I didn't want to deviate too much from the other images on tw.com. Maybe all of them should have a more descriptive title? Given that it *is* the documentation we're talking about, I think that would make sense. @Jermolene - any thoughts on this?
